src/controllers/albumController.js

```diff
@@ -0,0 +1,133 @@
+const uuidv4 = require("uuid/v4");
+const AlbumModel = require("../models/AlbumModel");
+const SongsModel = require("../models/SongsModel");
+
+class AlbumController {
+  getAlbums = async (req, res, next) => {
+    try {
+      const albums = await AlbumModel.findAll();
+      if (albums) {
+        res.status(200).json({ status: 200, albums });
+      } else {
+        res.status(404).json({ status: 404, error: "Albums not found" });
+      }
+    } catch (error) {
+      return next(error);
+    }
+  };
+
+  createAlbum = async (req, res, next) => {
+    const { name, review, price, artistId } = req.body;
+
+    try {
+      const newAlbum = await AlbumModel.create({
+        id: uuidv4(),
+        name,
+        review,
+        price,
+        artistId
+      });
+      if (newAlbum) {
+        res.status(201).json({
+          status: 201,
+          message: "successfully added new album",
+          album: newAlbum
```

Comment:
I'd suggest you change all your responses to have the following format:

```
res.status(200).json({
   status: 200,
   data: your_data,
   message: 'Succesfully, fetched album'
})
```

This makes it easier for the API consumer in the sense that he/she doesn't have to keep logging the request to see what the object key for a new request is, when you have the key as `album` then the next request has `songs` that brings confusion.